### ISSUE TITLE
Vollaphone focal OTA-3 : doesn't seem to work

### ISSUE THREAD

**Issue by gbdomubpkm** [Jakuko99/gpstoolkit#3]
Hello.
Thank you for the update to  focal.
Just tested on Volla: doen't seem to work.
logs https://dpaste.com/8NDAUCGNK
But is the first time i use your app : is there a procedure to use/test it ?
Regards

**Comment by Jakuko99** [comment]
Hello,
Thank you for your feedback, I will look into it, in the logs I didn't find anything that could cause the problem except missing library "libpq_cust.so", so maybe that could be the case. As for the question, there is no first time run procedure to run the app.

**Comment by gbdomubpkm** [comment]
Thank you.
Wait a new test .click.
Regards.

**Comment by Jakuko99** [comment]
@gbdomubpkm Try updating the app and running it again, hopefully I figured the issue correctly and app should work now

**Comment by gbdomubpkm** [comment]
I thought first  it doesn't work but YES IT WORKS ! Thank you !
I must test a little more however.

**Comment by Jakuko99** [comment]
That's great to hear! Then let me know if the GPS is working as well, if that works, then the rest should work without problems.

**Comment by gbdomubpkm** [comment]
I don't know if this can be improved in the app but, yes, the main difficulty is connecting to the GPS. I only managed it once, outside and in a town,  this afternoon.

**Comment by Jakuko99** [comment]
That is a global issue in UT, first fix takes some time and after that it works much faster. So there is no way of improving it in the app as assisted GPS needs to be implemented at system level, see this issue: https://gitlab.com/ubports/development/core/location-service/-/issues/13

**Comment by gbdomubpkm** [comment]
Yes, we need reliable GPS and for this issue to be resolved quickly.
